src/components/Organisms/MarketingPreferences/MarketingPreferences.js

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+
+import Text from '../../Atoms/Text/Text';
+import Link from '../../Atoms/Link/Link';
+import InputController from './InputController';
+import CheckAnswer from './CheckAnswer';
+import { Head, Body } from './MarketingPreferences.style';
+
+const MarketingPreferences = () => {
```

Comment:
I think it would be good to be able to have an initial values prop to pre-populate the fields. E.g. on the fundraise form, we already have their email address at this point, so would be good if they don't have to re-enter it